### ISSUE TITLE
Mac dependency scripts don't allow for quick easy installation.

### ISSUE THREAD

**Issue by asreimer** [vtsuperdarn/davitpy#122]
As @GPerry306 can confirm, the mac dependency scripts do not allow for a quick and easy installation of davitpy. Macports didn't seem to work at all, and then there were some issues with Homebrew as well.
![guidelines](https://cloud.githubusercontent.com/assets/4604819/5582593/a41313c4-9031-11e4-9660-4b6b6d274ee9.jpg)

I'm filling this issue just to say that the mac dependency scripts need some TLC and massaging. In linux, the dependency script works without hassle and installation is quick and simple. No reason that we can't have the same thing in OS X.

**Comment by aburrell** [comment]
I think another problem is that the install scripts assume that the person beginning to use davitpy is a new python user.  Perhaps it would be better/also useful to have a script that checks to see which dependencies (if any) need to be installed.  If we were really clever the check script could build a personalized install script.

**Comment by mrwessel** [comment]
I have a basic Python install (2.7.6) on my Mac, and now I will try to install davitpy.  Watch this space...

**Comment by johncoxon** [comment]
I managed to install it just fine using MacPorts. I installed the relevant bits and bobs, but to get `gfortran` and `mpif90` working I had to run `port select gcc` and `port select mpi`.

**Comment by mrwessel** [comment]
I'm using Homebrew.  The script worked ok, with the exception of being unable to find anything for PIL, until it got to Basemap.  There is no `cd` into the basemap directory before attempting to run the setup file so that failed.  When I did that job manually I also had to install geos (I think that was it, will check later) before Basemap would install.  Next was adding the `source profile.mac` to the bashrc; the script has it put into `.bash_profile` instead, but that doesn't exist so it failed.  Also I also had to set the `DAVITPY` variable manually.  I am still trying (off and on) to get mastermake to run properly, but there are permissions problems and I don't know yet what needs to be done as admin and what needs to be done as the normal user.
